tree/ntuple/v7/doc/BinaryFormatSpecification.md

```diff
@@ -802,6 +802,27 @@ This section is a comprehensive list of the C++ types with RNTuple I/O support.
 Within the supported type system complex types can be freely composed,
 e.g. `std::vector<MyEvent>` or `std::vector<std::vector<float>>`.
 
+### Type Name Normalization
+
+Type names are stored according to the following normalization rules
+  - All whitespaces are removed.
+  - CV qualifiers (`const`, `volatile`) are removed.
```

Comment:
Only the outer ones can be safely removed: `usertemplate<const int>` might have a complete different layout than `usertemplate<int>`

Reply:
Thanks, good point! Fixed.